collector/hwmon_linux.go

```diff
@@ -47,13 +51,18 @@ func init() {
 }
 
 type hwMonCollector struct {
-	logger log.Logger
+	deviceFilter deviceFilter
+	logger       log.Logger
 }
 
 // NewHwMonCollector returns a new Collector exposing /sys/class/hwmon stats
 // (similar to lm-sensors).
 func NewHwMonCollector(logger log.Logger) (Collector, error) {
-	return &hwMonCollector{logger}, nil
+
+	return &hwMonCollector{
+		logger:       logger,
+		deviceFilter: newDeviceFilter(*collectorHWmonUnitExclude, *collectorHWmonUnitExclude),
```

Comment:
```suggestion
		deviceFilter: newDeviceFilter(*collectorHWmonChipExclude, *collectorHWmonChipExclude),
```

Reply:
Doh! Hwmon subcomponents should be "chips", not "units". Updated the naming now.